doc/source/dev/core-dev/distributing.rst.inc

```diff
@@ -100,7 +100,7 @@ Building binary installers
 
    This section is only about building SciPy binary installers to *distribute*.
    For info on building SciPy on the same machine as where it will be used, see
-   `this scipy.org page <https://scipy.github.io/devdocs/dev/contributor/building.html>`_.
+   `this scipy.org page <https://scipy.github.io/devdocs/building/index.html#building-from-source>`_.
```

Comment:
This has a label https://github.com/scipy/scipy/blob/3643d5c37bc5c88273758a6e20c4b5034cec4b74/doc/source/building/index.rst?plain=1#L1
so hopefully we should be able to do 
```suggestion
   :ref:`_building-from-source`
```